src/terminal/interactive.rs

```diff
@@ -0,0 +1,26 @@
+// For interactively handling  reading in a string
+pub fn get_string(prompt_string: &str) -> String {
+    println!("{}", prompt_string);
+    let input: String = read!("{}\n");
```

Comment:
nit: i wonder if you can `read!("{} {}", prompt_string);

Reply:
it got mad, or i don't understand.